README.md

```diff
@@ -24,15 +24,16 @@ To further configure your plugin, use the following custom parameters in your `s
+| `flushMetricsToLogs` | Send custom metrics by using logs with the Datadog Forwarder Lambda function (recommended). Defaults to `true`. If you disable this parameter, it's required to set `apiKey` (or `apiKMSKey` if encrypted). Do not set `addExtension` to `true` when using `flushMetricsToLogs`.                                                                                                                                                            |
+| `site`               | Set which Datadog site to send data, this is only used when `flushMetricsToLogs` is `false` or `addExtension` is `true`. Possible values are `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com` and `ddog-gov.com`. The default is `datadoghq.com`.                                                                                                                                                                                                                                |
+| `apiKey`             | Datadog API Key, only needed when `flushMetricsToLogs` is `false` or `addExtension` is `true`. For more information about getting a Datadog API key, see the [API key documentation][3].                                                                                                                                                                                                                                    |
+| `apiKMSKey`          | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false` or `addExtension` is `true`, and you are using KMS encryption.                                                                                                                                                                                                                                             |
 | `addLayers`          | Whether to install the Datadog Lambda library as a layer. Defaults to `true`. Set to `false` when you plan to package the Datadog Lambda library to your function's deployment package on your own so that you can install a specific version of the Datadog Lambda library ([Python][4] or [Node.js][5]). |
+| `addExtension`    | Whether to install the Datadog Lambda Extension as a layer. Defaults to `false`. When enabled, it's required to set the `apiKey` (or `apiKMSKey`) parameter. While using `addExtension` set `flushMetricsToLogs` to `false` and leave out `forwarderArn`. Learn more about the Lambda Extension [here][8].                                                                                                                                                                                                                                                                      |
```

Comment:
I would add the note that the Lambda Extension is in preview (like we have in the CDK repo)